mysql-test/suite/rocksdb/t/collation.test

```diff
@@ -38,40 +38,39 @@ DROP TABLE t1;
 SET GLOBAL rocksdb_strict_collation_exceptions=t1;
 CREATE TABLE t1 (id INT primary key, value varchar(50), index(value)) engine=rocksdb charset utf8;
 DROP TABLE t1;
---error ER_UNSUPPORTED_COLLATION
 CREATE TABLE t2 (id INT primary key, value varchar(50), index(value)) engine=rocksdb charset utf8;
+DROP TABLE t2;
 
 # test regex for exception list
 SET GLOBAL rocksdb_strict_collation_exceptions="t.*";
 CREATE TABLE t123 (id INT primary key, value varchar(50), index(value)) engine=rocksdb charset utf8;
 DROP TABLE t123;
---error ER_UNSUPPORTED_COLLATION
 CREATE TABLE s123 (id INT primary key, value varchar(50), index(value)) engine=rocksdb charset utf8;
+DROP TABLE s123;
 
 SET GLOBAL rocksdb_strict_collation_exceptions=".t.*";
 CREATE TABLE xt123 (id INT primary key, value varchar(50), index(value)) engine=rocksdb charset utf8;
 DROP TABLE xt123;
---error ER_UNSUPPORTED_COLLATION
 CREATE TABLE t123 (id INT primary key, value varchar(50), index(value)) engine=rocksdb charset utf8;
+DROP TABLE t123;
 
 # test multiple entries in the list with vertical bar
 SET GLOBAL rocksdb_strict_collation_exceptions="s.*|t.*";
 CREATE TABLE s1 (id INT primary key, value varchar(50), index(value)) engine=rocksdb charset utf8;
 DROP TABLE s1;
 CREATE TABLE t1 (id INT primary key, value varchar(50), index(value)) engine=rocksdb charset utf8;
 DROP TABLE t1;
---error ER_UNSUPPORTED_COLLATION
 CREATE TABLE u1 (id INT primary key, value varchar(50), index(value)) engine=rocksdb charset utf8;
+DROP TABLE u1;
 
 # test allowing alters to create temporary tables
 SET GLOBAL rocksdb_strict_collation_exceptions='t1';
-CREATE TABLE t1 (id INT primary key, value varchar(50), index(value)) engine=rocksdb;
+CREATE TABLE t1 (id INT primary key, value varchar(50), index(value)) engine=rocksdb charset utf8;
```

Comment:
Maybe make the keyword case consistent too at the same time

Reply:
Yeah, true, this kind of stuff drives me nuts because the end result is that I end up re-working the entire test case to be consistent one way or another, then it destroys my ability to do clean merges :-|

Reply:
ugh. Do you what you have to do to stay sane

Reply:
Looking at the diffs on this test, we are pretty far diverged on the entire case as it is, so now I think fixing case consistency will not introduce any more merge issue than already exist in this test/result. So yeah, ignore my initial comment, I'll fix all of the case inconsistencies in this test with this PR and send it upstream at the next opportunity.